gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/HttpBasicPassTicketScheme.java

```diff
@@ -106,6 +107,7 @@ public AuthenticationCommand createCommand(Authentication authentication, AuthSo
         } catch (IRRPassTicketGenerationException e) {
             String error = String.format("Could not generate PassTicket for user ID %s and APPLID %s", userId, applId);
             logger.log(MessageType.DEBUG, error);
```

Comment:
is this other one not appearing in the spool?
Maybe let's keep only yours, this is redundant I think

Reply:
right, so why keep the one without details?

Reply:
Oh, I see. That makes sense. Thanks, updated.